client/src/lib/components/ui/Accordian/Accordian.svelte

```diff
@@ -0,0 +1,83 @@
+<script type="ts">
+    import downArrow from '$lib/assets/angle-down-solid.svg';
+
+    let acc: HTMLElement;
+
+    const expand = () => {
+        acc.classList.toggle("activeAccordian");
+        const panel = acc.nextElementSibling as HTMLElement;
+        if (!panel) return;
+        if (panel.style.maxHeight !== '0px') {
+            panel.style.maxHeight = '0px';
+        } else {
+            panel.style.maxHeight = `${panel.scrollHeight}px`;
+        }
+    }
+
+    export let title: string;
+    export let desc: string;
```

Comment:
use type definitions via interfaces as done in some of my components instead of defining them here

doing it that way helps in a mono repo structure, as we can import type definitions into the backed from that folder to maintain consistent request data

whether or not a ts backend is used is a separate issue, but our code should be best 🚀 👍

Reply:
Fixed the code quality: https://github.com/sohamjaiswal/apratim-shit/pull/40/commits/9bec6af790fe1b0777821f31d1327662771eb7de
No way the code quality will degrade.
thx btw.